docs/resources/Message.md

```diff
@@ -510,6 +517,23 @@ Additionally, the combined sum of characters in all `title`, `description`, `fie
 
 Embeds are deduplicated by URL.  If a message contains multiple embeds with the same URL, only the first is shown.
 
+#### Embed Fields by Embed Type
+
+Certain embed types are used to power special UIs. These embeds use [fields](#DOCS_RESOURCES_MESSAGE/embed-object-embed-field-structure) to include additional data in key-value pairs. Below is a reference of possible embed fields for each of the following embed types. 
+
+###### Poll Result Embed Fields
+
+| Embed Field Name              | Embed Field Value Description                              |
```

Comment:
Maybe a "parse as" (or something like that) column could be added to indicate what the value string will look like.
Additionally, a code sample would probably still help to visualize the explanation given above.